pipeline/reference.py

```diff
@@ -0,0 +1,156 @@
+'''
+Schema of subject information.
+'''
+import datajoint as dj
+
+schema = dj.schema('ttngu207_reference',locals())
+
+
+@schema
+class CorticalLayer(dj.Lookup):
+    definition = """
+    cortical_layer = 'N/A' : enum('N/A','1','2','3','4','5','6','2/3','3/4','4/5','5/6')
```

Comment:
This being a lookup layer, it doesn't make sense for the field data type to be an ENUM. Simply let this be a datatype like `varchar(8)` that can accommodate the content. Furthermore, for lookup tables, default value do not usually make sense.
```suggestion
    cortical_layer: varchar(8)    # layer within cortex
```